README.md

```diff
@@ -32,55 +34,14 @@ Extension configuration can be found from Magento administration interface, unde
 RSS-feed can be found from `store_url/sendsmaily/rss`. RSS-feed outputs 50 last edited products. **Only active and visible products are shown.**
 
 ## Manual export
+
 At any time you can trigger a manual export from `Newsletter` &rarr; `Newsletter subscribers` by selecting `Sendsmaily` from `Export to:` dropdown and hitting that `Export` button.
 
-To export specific subscriber(s), filter desired subscribers using the fields under the table header and click `Export`. 
+To export specific subscriber(s), filter desired subscribers using the fields under the table header and click `Export`.
 **NOTE! All filtered subscribers are exported. Selecting rows has no effect.**
 
 ## Troubleshooting
+
 **Regular export fails to run**
 
 Usually a good place to start would be to check Magento CRON's `Schedule Ahead for` value. We have found that value of **60** works the best, if you are running daily exports.
-
-## Changelog
-
-### 2.1.1
-
-- Add section to Admin panel for easy access to Smaily RSS-feed
-- RSS-feed now supports multistore
-
-### 2.1.0
-
-- Opt-in emails trigger for newsletter from subscribers
-- Opt-in trigger for subscribers changing their subscription state from settings page
-- Opt-out trigger for subscribers changing their subscription state from settings page
-- Google reCAPTCHA support for newsletter form
-- Bugfix - provide initial value for unsubscriber offset
-
-### 2.0.1
-
-- Add support for direct products import to the Smaily RSS-feed.
-
-### 2.0.0
-
-- Refractor functionality to implement new API
-- Customer Synchronization now removes unsubscribers from Magento store
-- Store unsubscribers are also updated in Smaily system
-- Use subdomain and username/password combination for authentication with Smaily
-- Updated admin page look
-
-## License
-```
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>
-```
```

Comment:
The license should be added as in other projects. 
https://github.com/sendsmaily/smaily-prestashop-module/blob/master/LICENCE.txt